### PR TITLE
feat: Add markpart API error handling.

### PR DESCRIPTION
**Pull request by defectiveAi**

### CODE REVIEW

File: apps/dh/api-dh/source/DataHub.WebApi/Clients/MarketParticipant/V1/ApiException.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    public IReadOnlyCollection<ApiErrorDescriptor> ApiErrors => ReadErrorResponse();
+
+    private IReadOnlyCollection<ApiErrorDescriptor> ReadErrorResponse()
+    {
+        if (_apiErrors != null)
+        {
+            return _apiErrors;
+        }
+
+        if (Response == null || StatusCode is not 400 and not 404)
```
Comment (mimse) [inline apps/dh/api-dh/source/DataHub.WebApi/Clients/MarketParticipant/V1/ApiException.cs:39]: Question: How about all the internal 500 errors?

File: libs/dh/globalization/assets-localization/src/assets/i18n/da.json [hunk 2]
```diff
@@ -512,8 +512,56 @@
...
   },
   "marketParticipant": {
-    "apiErrorCodes": {
-      "globalLocationNumberValidation": "'Aktørnummer' skal være et gyldigt GLN eller EIC nummer"
+    "market_participant": {
+      "error_fallback": "Computeren blev bedt om at udføre en handling, men gjorde det ikke. Årsagen var: {{ message }}",
+      "bad_argument": {
+        "missing_required_value": "Feltet '{{ param }}' kræver en værdi, men denne er ikke blevet angivet.",
+        "invalid_gln": "Værdien '{{ value.value }}' er ikke et gyldigt GLN- eller EIC-nummer.",
+        "invalid_length": "Indholdet af '{{ param }}' er for langt. Maksimal længde er {{ max }} tegn.",
```
Comment (mimse) [inline libs/dh/globalization/assets-localization/src/assets/i18n/da.json:520]: invalid_length can also be minimum length?

File: apps/dh/api-dh/source/DataHub.WebApi/Clients/MarketParticipant/V1/ApiException.cs (hunk 1 above)
Comment (defectiveAi) [reply apps/dh/api-dh/source/DataHub.WebApi/Clients/MarketParticipant/V1/ApiException.cs:39]: 500 errors do not return anything useful.

File: libs/dh/globalization/assets-localization/src/assets/i18n/da.json (hunk 2 above)
Comment (defectiveAi) [reply libs/dh/globalization/assets-localization/src/assets/i18n/da.json]: Fixed.

**Review by dzhavat — APPROVED** [review]
praise: beautiful 💯

(✅ on the frontend part)

**Pull request merged**